src/web/components/CardHeadline.tsx

```diff
@@ -52,6 +52,34 @@ const fontStyles = (size: SmallHeadlineSize) => {
 	}
 };
 
+const labTextStyles = (size: SmallHeadlineSize) => {
+	switch (size) {
+		case 'large':
+			return css`
+				${textSans.large()};
+				${until.desktop} {
+					${textSans.medium()};
+				}
+			`;
+		case 'medium':
+			return css`
+				${textSans.medium()};
+				${until.desktop} {
+					${textSans.small()};
+				}
+			`;
+		case 'small':
+			return css`
+				${textSans.small()};
+			`;
+		case 'tiny':
+			return css`
+				${textSans.xsmall()};
+				font-size: 14px;
+			`;
```

Comment:
Looking at the stories it seems some of these sizes are a bit off. 

We also need to set `{lineHeight: 'something'}` - maybe 'regular'?

Also looks like the standfirst needs the font setting. It might be easier to break these font changes out into a different PR/ticket?